tests/Microsoft.Identity.Web.Test.Common/TestHelpers/MsalTestTokenCacheProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Web.TokenCacheProviders;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Web.Test.Common.TestHelpers
+{
+    public class MsalTestTokenCacheProvider : MsalAbstractTokenCacheProvider
+    {
+        public IMemoryCache MemoryCache { get; set; }
```

Comment:
Please review visibility of all these members. I don't think a setter is needed.